.circleci/config.yml

```diff
@@ -31,10 +31,10 @@ jobs:
           command: docker login -u $DOCKER_USER -p $DOCKER_PASS
       - run:
           name: Test that Docker container works
-          command: docker run -it trussworks/ecs-service-logs:<< pipeline.git.tag >> --help
+          command: docker run -it trussworks/ecs-service-logs:${CIRCLE_SHA1} --help
       - run:
           name: Docker push
-          command: docker push trussworks/ecs-service-logs:<< pipeline.git.tag >>
+          command: docker push trussworks/ecs-service-logs:${CIRCLE_SHA1}
```

Comment:
One issue we'll see with this is that in docker hub we won't get tagged versions anymore. I'm not sure how to solve that. But at least you can test the code now.

Reply:
@mdawn - This change still has to be reverted. It did help us get past the local invocation of the `circleci` tool, but it will break our releases to docker hub. We'll have to revisit this in the future if we still want to make the `circleci` tool work.